Backtracking Algorithms

SubhradeepSS opened the pull request:
@Bhavnaharitsa please review my PR for the issue I created #66

Bhavnaharitsa commented on Algorithms/Backtracking/N Queen Problem.cpp:
```diff
@@ -0,0 +1,113 @@
+} 
+
+/* This function solves the N Queen problem using 
+Backtracking. It mainly uses solveNQUtil() to 
```
No need of this comments

Bhavnaharitsa commented on Algorithms/Backtracking/N Queen Problem.cpp:
```diff
@@ -0,0 +1,113 @@
+} 
+
+
+Output:
```
Comment output

Bhavnaharitsa replied:
Where is it? It is not

Bhavnaharitsa commented on Algorithms/Backtracking/N Queen Problem.cpp:
```diff
@@ -0,0 +1,113 @@
+/* A utility function to check if a queen can 
+be placed on board[row][col]. Note that this 
+function is called when "col" queens are 
+already placed in columns from 0 to col -1. 
```
Paste it on top with name of algorithm and description

SubhradeepSS replied:
@Bhavnaharitsa please review.
Any further changes required?

Bhavnaharitsa replied:
Do the changes mentioned below

SubhradeepSS commented:
@Bhavnaharitsa  updated.
Please suggest further changes.

Bhavnaharitsa commented on Algorithms/Backtracking/Hamiltonian Cycle.cpp:
```diff
@@ -0,0 +1,159 @@
+
+
+Output:
+Solution Exists: Following is one Hamiltonian Cycle
```
Comment output

SubhradeepSS replied:
Output is already in comment section at the end of the code

Bhavnaharitsa commented on Algorithms/Backtracking/M Coloring Problem.cpp:
```diff
@@ -0,0 +1,109 @@
+
+/* This function solves the m Coloring problem using Backtracking. 
+It mainly uses graphColoringUtil() to solve the problem. It returns 
+false if the m colors cannot be assigned, otherwise return true and 
```
no need explanation

Bhavnaharitsa commented on Algorithms/Backtracking/M Coloring Problem.cpp:
```diff
@@ -0,0 +1,109 @@
+} 
+
+
+Output:
```
comment output

SubhradeepSS replied:
Output is already in comment section at the end of the code

SubhradeepSS commented:
@Bhavnaharitsa  please review the new PR #67 ,not the old one

Bhavnaharitsa commented:
I am on pr #67 
Which one is the old one?

SubhradeepSS commented:
> I am on pr #67
> Which one is the old one?

Output is already commented out

SubhradeepSS commented:
Issue #71 fixed
Please review @Bhavnaharitsa

SubhradeepSS commented:
@Bhavnaharitsa  I think there is some issue;I updated the things on my repo.But its not showing.
I am closing this PR and opening a new one

SubhradeepSS commented:
> @Bhavnaharitsa I think there is some issue;I updated the things on my repo.But its not showing.
> I am closing this PR and opening a new one

@Bhavnaharitsa I squashed the earlier commits in which output was not commented.
U can check out in my master branch that I have updated

Bhavnaharitsa commented:
Conflicts

SubhradeepSS commented:
> Conflicts

Solved
